abdulrahman/style.css

```diff
@@ -0,0 +1,100 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: rgb(248, 217, 248);
+    line-height: 1.5;
+    word-spacing: 1.4px;
+    font-family: 'Roboto', sans-serif;
+    height: 100%;
+    color: black;
+    position: relative;
+}
+
+.card__container {
+    display: flex;
+    width: 60%;
+    height: auto;
+    margin: 300px auto;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    position: absolute;
+}
+
+.card {
+    border: 2px solid;
+    height: 500px;
+    margin: auto 10px;
+    border-radius: 15px;
+}
+
+.card__logo {
+    height: 45px;
+    width: 40px;
+    margin-left: 15px;
+    margin-top: 15px;
+    border-radius: 50%;
+}
+
+.card__image {
+    height: 190px;
```

Comment:
You height and width does not align and this makes the image looks skewed

Reply:
notes taken. will make sure to perfect everything just very soon